bgc_data/bgc_lfish_samples.dataresource.yaml

```diff
@@ -15,10 +15,6 @@ schema:
     type: integer
     required: true
     unique: true
-  - name: PROJECTNAME
-    title: "Source project name"
-    type: string
-    required: true   
```

Comment:
As with the other tables, PROJECTNAME, as well as LATITUDE, LONGITUDE, and STATIONNAME (below) are already in the `bgc_trip` table, so don't need to be included here.
@clairedavies This is assuming the Larval Fish products are all from (a subset of?) the same trips?

Reply:
The larval fish data includes non NRS data and so this information is required as they are NOT all from a subset of the same trips. There is some voyage information here that doesn't apply to the NRS. Where projectname = IMOS_NRS_xxx then the data is in the bgc_trip table, where projectname != IMOS_NRS_xxx then this information is not in the bgc_trip table.

Reply:
PROJECTNAME, LAT, LON and STATIONNAME add on [this commit](https://github.com/aodn/public-schema/pull/15/commits/4435f213a4cd7f41af3840167afa0b8cf7185cf8)

Reply:
Thanks for clarifying @clairedavies. I'm happy with that, if you think that's the easiest way to manage these data.
Alternatively, couldn't we just add all the Larval Fish trips into the `bgc_trip` table? There are already plenty of non-NRS trips in there!
Or is that more complicated at your end?

Reply:
I prefer it that way as a) they are voyage data not station data, b) they are only relevant to fish, nothing else, c) the fish can then be stand alone as a product for all their data which is what Iain wants, d) they are finishing soon and therefore once that is done we can move on.